clouddriver-azure/src/main/groovy/com/netflix/spinnaker/clouddriver/azure/resources/servergroup/cache/AzureServerGroupCachingAgent.groovy

```diff
@@ -30,7 +30,7 @@ import com.netflix.spinnaker.clouddriver.azure.common.AzureUtilities
 import com.netflix.spinnaker.clouddriver.azure.common.cache.AzureCachingAgent
 import com.netflix.spinnaker.clouddriver.azure.common.cache.MutableCacheData
 import com.netflix.spinnaker.clouddriver.azure.resources.common.cache.Keys
-import com.netflix.spinnaker.clouddriver.azure.resources.common.cache.Keys.Namespace
+import static com.netflix.spinnaker.clouddriver.azure.resources.common.cache.Keys.Namespace.*
```

Comment:
It is required to access static members of a class. Since Namespace is a static enum, we have to add this to reference them directly (i.e., not have to preface everything with 'Key.Namespace')